index.md

```diff
@@ -16,49 +16,33 @@ order: 1
 ## Introduction
 {:#introduction}
 
-Web accessibility evaluation tools are software programs or online services. They help determine if a Web site is accessible. This document highlights features of evaluation tools. It discusses different aspects of evaluation tools to consider when selecting these tools. 
-
-Automated web accessibility evaluation tools are unable to test all accessibility criteria. Many aspects of accessibility need human verification. 
-
-The [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/) provides several features of evaluation tools. Some of these features include: 
-
-- Guidelines
-- Specific uses of the tool
-- What authoring tools integrate with the evaluation tool
-- API
-- Browser plugin
-- etc.
-
-This document also discusses some specific usage of evaluation tools, such as:
-- Reports
-- Step-by-step evaluation
-- In-page feedback
-- Page transformation
-
-Finally, this document offers further aspects to consider, such as:
-- Organizational structure and development process
-- Complexity and size of Web site
-- Skills and knowledge of Web developers
-
-*Disclaimer*: WAI encourages the development and evolution of Web accessibility evaluation tools. WAI maintains an extensive [list of evaluation, repair and transformation tools](https://www.w3.org/WAI/ER/tools/). WAI does not endorse or promote any single tool or vendor.
+Web accessibility evaluation tools are software programs or online servicesthat help determine if a Web site is accessible. This document highlights features of evaluation tools, and discusses different aspects to consider when selecting these tools. Links to different tools is provide in the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/).
 
 ## What to Expect from Evaluations Tools
 {:#Expect}
 
-Web accessibility evaluation tools can be a powerful part of a developer's arsenal. You can use evaluation tools through all phases of Web site development. They assist with determining the level of accessibility of Web sites. Evaluation tools can verify the conformance of Web sites for items that can be reviewed automatically. They can also help with manual reviews.
+Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools can provide fully-automated checks, as well as help you with manual review.
```

Comment:
Intent is to quickly clarify that "tools" is not limited to "developers" and "automated checking", which is unfortunately a common misconception.

Reply:
+1